tools/claude-autoresume/verify/agent-test.sh

```
@@ -0,0 +1,59 @@
+chmod 600 "$HOME/.config/claude-autoresume/sessions.json"
+
+echo "===== A) doctor on clean machine ====="
+if "$BIN" doctor 2>&1 | tee /tmp/doctor.out | tail -3; then :; fi
+grep -q 'doctor: PASS' /tmp/doctor.out && mark "DOCTOR_PASS" || { mark "DOCTOR_FAIL"; fail=1; }
+
+echo "===== A) direct launcher start (no systemd) ====="
+"$BIN" start 2>&1 | tee /tmp/start.out
+grep -q 'done: enabled=1 launched=1 live=1' /tmp/start.out && mark "DIRECT_LAUNCH_OK" || { mark "DIRECT_LAUNCH_FAIL"; fail=1; }
+if ! grep -q '\[!!\]' /tmp/start.out; then mark "NO_FALSE_WARN"; else mark "UNEXPECTED_WARN"; fail=1; fi
+echo "--- live sessions ---"; tmux -L claude-autoresume ls 2>&1
+tmux -L claude-autoresume capture-pane -p -t car-proj 2>/dev/null | grep -qi 'STUB CLAUDE RESUMED' && mark "PANE_RESUMED" || mark "PANE_EMPTY"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)